n2y/notion.py

```diff
@@ -116,6 +118,7 @@ def __init__(
 
         self.databases_cache = {}
         self.pages_cache = {}
+        self.yaml_cache = {}
```

Comment:
I think we should be able to avoid the need for another cache, since we can instead rely on the databases_cache.